ibmcloud/image/verify.sh

```diff
@@ -54,6 +54,10 @@ while ! ibmcloud is instance $id --output json | jq -e '.status == "running"' >
     sleep 5
 done
 
+skip_verify_console=$SKIP_VERIFY_CONSOLE
+
+if ["$skip_verify_console" = false] ; then
+
```

Comment:
Could you change the code to skip console checking only when the variable is set?

The code will be something like this.

```
if [[ -n "${SKIP_VERIFY_CONSOLE:-}" ]]; then
	echo 'SKIP_VERIFY_CONSOLE is set. skip console checking'
	exit 0
fi
```

`"${SKIP_VERIFY_CONSOLE:-}"` will suppress the "unbound variable" error, even when an older version of bash is used.